src/nl/rubensten/texifyidea/settings/TexifyConfigurable.kt

```diff
@@ -1,34 +1,42 @@
 package nl.rubensten.texifyidea.settings
 
 import com.intellij.openapi.options.SearchableConfigurable
+import nl.rubensten.texifyidea.settings.labeldefiningcommands.TexifyConfigurableLabelCommands
 import java.awt.FlowLayout
 import javax.swing.BoxLayout
 import javax.swing.JCheckBox
+import javax.swing.JComponent
 import javax.swing.JPanel
 
 /**
  * @author Ruben Schellekens, Sten Wessel
  */
 class TexifyConfigurable(private val settings: TexifySettings) : SearchableConfigurable {
-
     private lateinit var automaticSoftWraps: JCheckBox
     private lateinit var automaticSecondInlineMathSymbol: JCheckBox
     private lateinit var automaticUpDownBracket: JCheckBox
     private lateinit var automaticItemInItemize: JCheckBox
+    private lateinit var table: TexifyConfigurableLabelCommands
```

Comment:
Rename to something more sensible, like `labelDefiningCommands` for example.

Reply:
fixed in commit ed34f48cd6dd8da4a8ed3d61d6cf44599f4bc7dd